.github/workflows/build.yml

```diff
@@ -5,9 +5,14 @@ on:
     types: [opened, synchronize, reopened, edited, ready_for_review, labeled]
 
 jobs:
-  chart-certification:
-    name: Chart Certification
+  setup:
+    name: Setup CI
     runs-on: ubuntu-22.04
+    outputs:
+      run_build: ${{ steps.check_build_required.outputs.run-build }}
+      verifier-action-image: ${{ steps.set-env.outputs.verifier-action-image }}
+      insecure_skip_tls_verify: ${{ steps.set-env.outputs.insecure_skip_tls_verify }}
```

Comment:
Can we normalize these outputs to use either hyphens (e.g. `run-build`) or camel case to divide words?

I'd say you don't have to modify the steps themselves now, because that will most likely require updating scripts. Leave those be, but when linking steps to outputs, lets normalize.

Reply:
Might go without saying, but remember that changing these outputs implies changing references in other steps.

Reply:
I will do this in a separate pull request since it requires changing the Python scripts that are updating the variables